cmd/ui/src/views/EarlyAccessFeatures/EarlyAccessFeatures.tsx

```diff
@@ -163,6 +159,12 @@ const EarlyAccessFeatures: React.FC = () => {
                                     <EarlyAccessFeatureToggle
                                         flag={flag}
                                         onClick={(flagId) => {
+                                            // TODO: Consider adding flag keys to cue files so we don't use random strings. Consider adding more flexibility/composability to side effects for toggling feature flags on and off
```

Comment:
Can we author a ticket for this and remove this TODO?

Reply:
I'll author a ticket a for it 👍 Though I do like having the TODO here as hopefully it informs others in the area that this is a pattern that we do not want to continue without having to be aware that there is a ticket to make this change.

Reply:
Okie dokie!

Reply:
feature flags in cue: BED-4583

Comment is updated and moved in changeset: https://github.com/SpecterOps/BloodHound/pull/717/files.